### ISSUE TITLE
Out of step with EFF website

### ISSUE THREAD

**Issue by ghost** [EFForg/https-docs#2]
The HTTPS Everywhere documentation at https://www.eff.org/https-everywhere is out of sync with this Git repository. For example, the current version of the "Development" page [in this repo](https://github.com/EFForg/https-docs/blob/236ea7c5f0404a2d0f4b7ac1cf614b555ae734d4/development.html) says:

> Please work off of the "3.0" branch if you're submitting changes to the latest stable release and use "master" if you're submitting changes to the latest development release.

However, that text is not present in the corresponding page on the EFF website: https://www.eff.org/https-everywhere/development

Please could the EFF website's documentation pages for HTTPS Everywhere be automatically built from this Git repository (or [better still](https://github.com/EFForg/https-docs/issues/2), the code repository), or otherwise kept in sync with it?

**Comment by brainwane** [comment]
You're right -- thanks, @sampablokuper .

As you suggested here and in #3, we want to close and archive this repo, and de-duplicate the docs in [EFForg/https-everywhere](https://github.com/EFForg/https-everywhere/), and welcome your continued participation there. So I'm going to put a bit of time into figuring this out, porting useful prose from these documents into https://github.com/EFForg/https-everywhere/tree/master/docs/en_US/ , before we archive this repository. I'd welcome your help.

**Comment by ghost** [comment]
@brainwane , thanks for your kind message; much appreciated.

Will file PRs if I get a chance, but please don't wait on me: I might not get to this before you've already taken care of it. I trust your judgement!

**Comment by brainwane** [comment]
It sounds like https://github.com/EFForg/https-everywhere/pull/14423 is going to be a step on the way to you making a good diff between these two sets of docs, @sampablokuper -- is that right? Thanks!

**Comment by ghost** [comment]
@brainwane wrote:
> It sounds like EFForg/https-everywhere#14423 is going to be a step on the way to you making a good diff between these two sets of docs, @sampablokuper -- is that right?

Spot on :)

**Comment by ghost** [comment]
Started to diff these just now. Will check them off as time permits.

- [x] deploying-https.md
- [x] development.md
- [x] faq.md
- [x] rulesets.md

**Comment by ghost** [comment]
@brainwane , in relation to
> porting useful prose from these documents into https://github.com/EFForg/https-everywhere/tree/master/docs/en_US/

these are my findings:

- `deploying-https.md` is not in https://github.com/EFForg/https-everywhere/tree/master/docs/en_US/ at all. I have created a PR [here](https://github.com/EFForg/https-everywhere/pull/14787) to add it to that repo (which can of course be declined if this file's absence from that repo is intentional). **As such, the copy here is ready to be archived.**
- `development.md` gained a typo since it was migrated from EFForg/https-docs to EFForg/https-everywhere. I have generated a PR to fix this [here](https://github.com/EFForg/https-everywhere/pull/14786). **As such, the copy here is ready to be archived.**
- `faq.md` in this repo contains nothing that isn't already covered at least as well at https://github.com/EFForg/https-everywhere/tree/master/docs/en_US/ . **As such, the copy here is ready to be archived.**
- `rulesets.md` here contains some prose that is not present in the copy under EFForg/https-everywhere. I have added a PR to address that [here](https://github.com/EFForg/https-everywhere/pull/14788), which can of course be declined if that prose is no longer wanted. **As such, the copy here is ready to be archived.**

**tl;dr:** this repo can now be archived. :+1:

**Comment by Hainish** [comment]
@sampablokuper thanks for looking into this.  I don't have ownership of this repo, I've requested from the organization owners that it be archived.
